lib/house.rb

```diff
@@ -0,0 +1,50 @@
+class House
+
+  attr_reader :price, :address, :rooms, :details
+
+  def initialize(price, address)
+    @price = price.gsub("$", "").to_i
+    @address = address
+    @rooms = []
+    @details = {
```

Comment:
Since we are already storing `@price` and `@address`, we don't need to store the details on the object, this could be a standalone method to cut down on memory used.